src/utils/job-socket.js

```diff
@@ -8,41 +8,41 @@ export default class JobSocket extends EventEmitter {
         // setup websocket
         this._socket = new WebSocket(url);
 
-        this._socket.onopen = (event) => { this.emit('open') };
-        this._socket.onclose = (event) => { this.emit('close') };
+        this._socket.onopen = () => { this.emit("open"); };
```

Comment:
are you removing `event` here because eslint complains about having unused arguments?